packages/common/src/serverHost.ts

```diff
@@ -0,0 +1,6 @@
+export const hostDefined = !!globalThis.process.env['VITE_SERVER_HOST']
```

Comment:
we have these already in client-core/src/util/config.ts lets unify these into a common, as this file has done here